### DISCUSSION TITLE
Ruby on rails technical question.

### DISCUSSION THREAD

**Discussion by greatgeneral94** [community/community#20804] [Discussions] [answered]
What does garbage collection do in Ruby on Rails?
If anyone know about this, please help me.

**Comment by ghost** [comment, marked as answer]
Garbage collection performs a variety of functions, including allowing for pointer values removal and removing the inaccessible objects from the memory, among other things. In essence, garbage collection helps free up memory for other processes.

**Reply by greatgeneral94** [reply]
Thank you.

**Reply by devcrafta** [reply]
Good
